### ISSUE TITLE
Pima Indians Diabetes dataset accuracy with AutoSKLearn is 55%?

### ISSUE THREAD

**Issue by giorgio79** [automl/auto-sklearn#29]
Just ran this snippet to test the accuracy of results with autosklearn and got a terrible 55%. Any tips why? :)

>import numpy as np
import urllib
import autosklearn.classification
import sklearn.datasets
url = "http://goo.gl/j0Rvxq"
raw_data = urllib.urlopen(url)
dataset = np.loadtxt(raw_data, delimiter=",")
print(dataset.shape)
X = dataset[:,0:7]
y = dataset[:,8]
indices = np.arange(X.shape[0])
np.random.shuffle(indices)
X = X[indices]
y = y[indices]
X_train = X[:700]
y_train = y[:700]
X_test = X[700:]
y_test = y[700:]
automl = autosklearn.classification.AutoSklearnClassifier()
automl.fit(X_train, y_train)
print(automl.score(X_test,y_test))`

**Comment by mfeurer** [comment]
Hi, the score functions currently scale everything in the range of [score_of_random_predictions, 1]. Because your dataset is a binary dataset and the score of a random prediction would be 0.5, it actually means that you have an accuracy of 75%. You can use the methods of scikit-learn to verify that.

Besides that, auto-sklearn does not use cross-validation by default, but holdout. In the case of your dataset, which is rather small, it uses only 466 data points to train the models, and the other 234 data points to optimize hyperparameters and build an ensemble. 

Using CV is not yet straight-forward, but we will add an example of that in the future (thus I'm leaving this issue open).

**Comment by mfeurer** [comment]
Closing this because this issue stalled - please reopen if this problem still persists with the latest version of auto-sklearn.
